torch2trt/converters/interpolate.py

```diff
@@ -86,7 +86,11 @@ def convert_interpolate_trt7(ctx):
         layer.resize_mode=trt.ResizeMode.NEAREST
 
     if align_corners != None:
-        layer.align_corners = align_corners
+        if trt_version() > '8.0':
+            if align_corners:
```

Comment:
Just out of curiosity, since we already perform the following check: 

```
if align_corners != None
```
Wouldn't the following `if align_corners` be redundant?

Reply:
Hi @JWLee89,  thanks for pointing this out. I'm not sure if it happens in practice, but I think if the user passes ``False`` the second check is needed.  There might be a cleaner way to organize this logic.  Let me know if you see any issues.

Reply:
Hi @jaybdub ,

Thank you for the response. 

If the goal is to filter out `None` and `False`,  I think we can replace the following code block 

```python
if align_corners != None:
    layer.align_corners = align_corners
    if trt_version() > '8.0':
        if align_corners:
```

With the following 

```Python
# replace if align_corners != None to filter out both None and False
if align_corners:
    layer.align_corners = align_corners
    if trt_version() > '8.0':
        # enter remaining logic ...
```

The reason is because `if align_corners` automatically filters out values that are `None` and `False`. I have added a simple code snippet below to prove that. 

```Python
    for align_corners in [False, None, True]:
        # align corners filters out both False and None
        if align_corners:
            # Truthy only called on "True"
            print(f"Truthy! Passed in: {align_corners}")
        else:
            print(f"Falsy! Passed in: {align_corners}")
```

Of course, I might have missed out on some internal implementation detail-specific use case or taken things out of context. If so, please feel free to disregard this, as removing / adding a for loop is not too critical. 

Thank you for taking time to read through this!